src/TLabel.cpp

```diff
@@ -272,22 +264,13 @@ bool TLabel::forwardEventToMapper(QEvent* event)
     return false;
 }
 
-// This function iterates through the provided event parameters,
-// searching for parameters that are references to values in the
-// Lua registry, and correctly dereferences them. This allows
-// the parameters to be safely overwritten.
-void TLabel::releaseParams(TEvent& params)
+// This function deferences previous functions in the Lua registry.
+// This allows the functions to be safely overwritten.
+void TLabel::releaseFunc(const int mfunc, const int func)
```

Comment:
I'd like to see better naming here - when you look at this function signature and parameters, it doesn't tell you a whole lot.

```suggestion
void TLabel::releaseFunction(const int existingFunction, const int newFunction)
```